src/main/java/org/humancellatlas/ingest/submission/SubmissionEnvelope.java

```diff
@@ -30,8 +30,6 @@
     private SubmissionState submissionState;
     private SubmissionGraphValidationState graphValidationState;
     private @Setter
-    List<GraphValidationError> graphValidationErrors;
```

Comment:
why remove this?

Reply:
Errors are now on the entities that caused them